example/spec/syncGranule/SyncGranuleSuccessSpec.js

```diff
@@ -0,0 +1,78 @@
+const fs = require('fs');
+const path = require('path');
+const { s3, s3ObjectExists } = require('@cumulus/common/aws');
+const { buildAndExecuteWorkflow, LambdaStep } = require('@cumulus/integration-tests');
+
+const { loadConfig, templateFile } = require('../helpers/testUtils');
+const config = loadConfig();
+const lambdaStep = new LambdaStep();
+const taskName = 'SyncGranule';
+
+const outputPayloadTemplateFilename = './spec/syncGranule/SyncGranule.output.payload.template.json'; // eslint-disable-line max-len
+const templatedOutputPayloadFilename = templateFile({
+  inputTemplateFilename: outputPayloadTemplateFilename,
+  config: config.SyncGranule
+});
+const expectedPayload = JSON.parse(fs.readFileSync(templatedOutputPayloadFilename));
+
+
+describe('The Sync Granules workflow', () => {
+  const inputPayloadFilename = './spec/syncGranule/SyncGranule.input.payload.json';
+  const inputPayload = JSON.parse(fs.readFileSync(inputPayloadFilename));
+  const collection = { name: 'MOD09GQ', version: '006' };
+  const provider = { id: 's3_provider' };
+  let workflowExecution = null;
+
+  beforeAll(async () => {
+    // eslint-disable-next-line function-paren-newline
+    workflowExecution = await buildAndExecuteWorkflow(
+      config.stackName, config.bucket, taskName, collection, provider, inputPayload
+    );
+  });
+
+  it('completes execution with success status', () => {
+    expect(workflowExecution.status).toEqual('SUCCEEDED');
+  });
+
+  describe('the SyncGranule Lambda function', () => {
+    let lambdaOutput = null;
+    let files;
+    let key1;
+    let key2;
+    const existCheck = [];
+
+    beforeAll(async () => {
+      lambdaOutput = await lambdaStep.getStepOutput(workflowExecution.executionArn, 'SyncGranule');
+      files = lambdaOutput.payload.granules[0].files;
+      key1 = path.join(files[0].fileStagingDir, files[0].name);
+      key2 = path.join(files[1].fileStagingDir, files[1].name);
+      existCheck[0] = await s3ObjectExists({ Bucket: files[0].bucket, Key: key1 });
+      existCheck[1] = await s3ObjectExists({ Bucket: files[1].bucket, Key: key2 });
+    });
+
+    afterAll(async () => {
+      await s3().deleteObject({ Bucket: files[0].bucket, Key: key1 }).promise();
+      await s3().deleteObject({ Bucket: files[1].bucket, Key: key2 }).promise();
+    });
+
+    it('has expected payload', () => {
+      expect(lambdaOutput.payload).toEqual(expectedPayload);
+    });
+
+    it('has expected updated meta', () => {
+      expect(lambdaOutput.meta.input_granules).toEqual(expectedPayload.granules);
+    });
+
+    it('files receive custom staging directory', () => {
+      files.forEach((file) => {
+        expect(file.fileStagingDir).toMatch('custom-staging-dir\/.*');
+      });
+    });
+
+    it('files exist in staging location', () => {
```

Comment:
Pure verbiage thing... should be something like "adds files to the staging location", verb first.